config/tgl-cavs.toml

```diff
@@ -286,3 +286,23 @@ count = 10
 
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
+        [[module.entry]]
```

Comment:
do we not have the ability to import a "common" file?

Reply:
what's the common part ? Thanks!

Reply:
The module itself, i see this file is for tgl only